### PR TITLE
DOCS-5857 button label properties

### PR DESCRIPTION
**Pull request by wwarner-salesforce**

### CODE REVIEW

**Review by spezzirriemiliano — APPROVED** [review]

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 1]
```diff
@@ -32,6 +32,8 @@ The API card component shows information about an API, including icon, name, and
 * *Card Border*: Set the thickness of the card borders.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:32]: Going forward, let's use definition list format for this, or at least a linebreak after the initial bullet fragment or sentence. See style reference.

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 2]
```diff
@@ -32,6 +32,8 @@ The API card component shows information about an API, including icon, name, and
...
 * *Card Border Radius*: Set the radius of the card borders. A larger radius makes the cards look more rounded.
 * *Card Shadow*: Set the shadow beneath the card.
+* *API Details Button Label*: Set the label of the API Details button. Default is `API Details`.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]: GLOBALLY remove backtick around GUI labels; that is reserved for syntax.

See https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 3]
```diff
@@ -48,9 +50,11 @@ The API carousel component shows a horizontally scrollable set of API cards.
 * *Card Border*: Set the thickness of the card borders.
 * *Card Border Radius*: Set the radius of the card borders. A larger radius makes the cards look more rounded.
 * *Card Shadow*: Set the shadow beneath each card.
-* *Message If Empty*: Show this message if the carousel has no APIs. Default is "There are no APIs available right now."
+* *Message If Empty*: Show this message if the carousel has no APIs. Default is `There are no APIs available right now.`
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]: Show a message

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 4]
```diff
@@ -48,9 +50,11 @@ The API carousel component shows a horizontally scrollable set of API cards.
...
 * *Filter Condition (Advanced)*: Filter APIs using package names, field names such as "API name" or "API description", and comparison of strings to those fields.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]: "API description,"

GLOBALLY put commas and periods within quotation marks unless they might be confused as part of the quoted material (rare).

**Review by kbostrom — APPROVED** [review]
Here you go!

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:32]: Good point-- that is better.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 2 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Confusingly, this is the value the user types into the interface, which is why it has backticks. Then the system produces a web page with a button that displays that value as a GUI label.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:32]: I will make a new PR to change this file to definition list format.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 4 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Fixed.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 3 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Good point. Fixed.

**Pull request merged**